sharedbuffers/pool.py

```diff
@@ -66,13 +106,19 @@ def __init__(self, section_size=DEFAULT_SECTION_SIZE, temp_kwargs={}, idmap_kwar
 
     @property
     def size(self):
+        """
```

Comment:
Doesn't the code style allow for these short comments to be one lined? `""" short comment """ ?

Reply:
Yes... but it doesn't force you, and I don't like it.